include/lbann/models/recurrent.hpp

```diff
@@ -26,6 +26,8 @@
 // recurrent .hpp .cpp - Recurrent neural network models
 ////////////////////////////////////////////////////////////////////////////////
 
+#if 0
```

Comment:
Why is this disabled?

Reply:
I forgot to uncomment, although I think we might want to eliminate the recurrent model. I've opened up issue #467.

Reply:
I think we should uncomment or address #467 first.

Reply:
Recurrent models are removed in #471.